examples/singlet-triplet-fission/README.md

```diff
@@ -5,16 +5,16 @@ https://github.com/SMTG-UCL/singlet-fission-screening
 
 * step 0: select a subset
 
-frame_select.py -fxyz all.xyz -y tda_triplet --n 1000 --algo random
+python ../../scripts/frame_select.py -fxyz all.xyz -y tda_triplet --n 1000 --algo random
 
 * step 1: generate the kernel matrix
 
-gen_soap_kmat.py -fxyz ASAP-random-n-1000.xyz --rcut 2.5 --n 4 --l 6 --g 0.4 --periodic False --plot True
```

Comment:
We decided to just use the name of the script, instead of the relative path in the examples,
because we expect people to install it